### PR TITLE
Update UploadField.php to allow configurable GridFieldPaginator

### PR DESCRIPTION
**Pull request by Legin76**
Changed $config->addComponent(new GridFieldPaginator(11)); to allow it to be configured from the config.yml

### CODE REVIEW

**Comment by dhensby** [comment]
I can merge this locally to fix the indents. Thanks and welcome to the glorious world of open source contribution!

File: forms/UploadField.php [hunk 1]
```diff
@@ -1498,6 +1498,9 @@ class UploadField_SelectHandler extends RequestHandler {
 	 */
 	protected $folderName;
 
+	// Set pagnation quantity for file list field
```
Comment (tractorcow) [inline forms/UploadField.php]: Can you please use phpdoc for this. E.g.

```php
/**
 * Set pagination quantity for file list field
 * 
 * @config
 * @var int
 */
```

**Comment by dhensby** [comment]
THANKS!

File: forms/UploadField.php (hunk 1 above)
Comment (Legin76) [inline forms/UploadField.php]: Changed.. I hope I've done it correctly.

Comment (tractorcow) [inline forms/UploadField.php]: Er, you didn't actually change it though? did you make the change locally but forget to push it up?

Comment (kinglozzer) [inline forms/UploadField.php]: S’okay - Dan did it in the merge commit https://github.com/silverstripe/silverstripe-framework/commit/31a4e184f737ad4f23d3c15b80199198c60a17f0

Comment (dhensby) [inline forms/UploadField.php]: Yes, I did it as part of the merge. so I merged it before it got pushed

**Pull request merged**